### PR TITLE
I complete login feature so please merge into master

### PR DESCRIPTION
**Pull request by chiragkhimani**
Pls merge into master

### CODE REVIEW

**Comment by chiragkhimani** [comment]
Well Done!

**Comment by IrinaSla** [comment]
nice work

**Comment by azizimisri** [comment]
Good job!

File: src/com/automation/git/GitPractice.java [hunk 1]
```diff
@@ -2,12 +2,7 @@
...
 public class GitPractice {
 	public static void main(String[] args) {
-		System.out.println("From Chirag PC");
-		System.out.println("From Chirag Again");
-		System.out.println("From Aziz PC");
-		System.out.println("Hello From Chirag");
-		System.out.println("Hello class");
-		System.out.println("Hello from Irina");
-	
+		System.out.println("Master");
```
Comment (chiragkhimani) [inline src/com/automation/git/GitPractice.java:5]: Why you not declare a variable? this is bad practice

**Pull request merged**